app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command):
+    with open(command[3:12], 'r') as old, open(command[12:], 'a') as new:
```

Comment:
Will slices work with filenames of different lengths? 
I suppose, won't. Use `split` to get filenames